design/gateway-apis-implementation.md

```diff
@@ -280,12 +280,10 @@ Contour’s implementation of the Operator has taken a similar approach where ba
 
 Initially Contour can do the same work to move that same logic and expose Envoy via annotations on the Service Type=LoadBalancer.
 
-The future looking way to approach this is to utilize GatewayAPI fully and instead of exposing a service with type=LoadBalancer, Contour can create a GatewayClass, Gateway which selects a TCPRoute used to send traffic to the Envoy fleet.
-The details defined in the Contour configuration CRD will be used to pre-populate annotations that will be stored in a Kubernetes configmap and referenced from the GatewayClass.Spec.ParametersRef.
-The Gateway Stub controller will be utilized to configure a new Kubernetes Service with the annotations defined in the configmap on the GatewayClass and send traffic to Envoy.
-Once implementers catch up, specific implementations will be utilized and the stub controller can be deprecated.
+Additionally, Contour will support utilizing GatewayAPI itself fully and instead of exposing a service with type=LoadBalancer, Contour can create a GatewayClass, Gateway which selects a TCPRoute used to send traffic to the Envoy fleet.
```

Comment:
Nit: "Eventually" is probably closer to what we mean here, since we're not going to do this right away, I think.

Reply:
Yup, it's just down to timing I think. I'd like to have the config option in the configuration file. If that's set then Contour doesn't manage the envoy service of type=LoadBalancer, just a regular ClusterIP service.